src/components/modal-container/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { styled } from "@stitches/react";
+
+const Container = styled("div", {
+  background: "$bg1",
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "center",
+  border: "1px solid $border",
+  borderRadius: "20px",
+  padding: "30px",
+  maxWidth: "420px",
+  position: "relative",
+  boxSizing: "border-box",
+});
+
+const Header = styled("div", {
+  fontFamily: "sans-serif",
+  fontWeight: "700",
+  fontSize: "20px",
+  lineHeight: "24px",
+  color: "$text",
+  display: "flex",
+  alignItems: "center",
+
+  marginBottom: "40px",
+  // position: "relative",
+});
+
+const CloseButton = styled("button", {
+  border: "none",
+  background: "none",
+  cursor: "pointer",
+  outline: "none",
+  position: "absolute",
+  right: "30px",
+});
+
+interface ModalContainerProps {
+  headerText?: string;
+  children: React.ReactNode;
+}
+
+export const ModalContainer: React.FC<ModalContainerProps> = ({
+  headerText = "How was your onboarding?",
+  children
+}) => {
+  return (
+    <Container>
+      <Header>
+        {headerText}
+        <CloseButton aria-label="Close Feedback Modal">
+          <img src="/icons/x-icon.svg" height="16px" width="16px" />
```

Comment:
It's always better to inline icons because this will cause an extra network request (because the browser thinks this is an image) so the image will load a bit later and cause some delay, etc.

Reply:
I seem I will make it a component then.